openedx_ledger/admin.py

```diff
@@ -231,7 +231,8 @@ class Meta:
 
     quantity_usd_input = forms.FloatField(
         required=True,
-        help_text='Amount of adjustment in US Dollars.',
+        help_text='Amount of adjustment in US Dollars. If the adjustment reduces the total deposits, please verify \
+        that the policies spend limit associated to this subsidy does not exceed the new total deposits after this adjustment is made.',
```

Comment:
nit: is the `spend limit` belong to the policies? If so, should the "policies" be possessive as `policies'`?

```suggestion
        that the policies' spend limit associated to this subsidy does not exceed the new total deposits after this adjustment is made.',
```